src/components/shared/Entity/ValidationErrorSummary/materialization/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { Alert, AlertTitle, Collapse } from '@mui/material';
+import MessageWithLink from 'components/content/MessageWithLink';
+import DetailsErrors from 'components/shared/Entity/ValidationErrorSummary/DetailsErrors';
+import EndpointConfigErrors from 'components/shared/Entity/ValidationErrorSummary/EndpointConfigErrors';
+import NoConnectorError from 'components/shared/Entity/ValidationErrorSummary/NoConnectorError';
+import ResourceConfigErrors from 'components/shared/Entity/ValidationErrorSummary/ResourceConfigErrors';
+import useGlobalSearchParams, {
+    GlobalSearchParams,
+} from 'hooks/searchParams/useGlobalSearchParams';
+import { FormattedMessage } from 'react-intl';
+import { useFormStateStore_displayValidation } from 'stores/FormState';
+import {
+    useResourceConfig_hydrationErrorsExist,
+    useResourceConfig_resourceConfigErrorsExist,
+} from 'stores/ResourceConfig';
+import { hasLength } from 'utils/misc-utils';
+
+interface Props {
+    errorsExist: boolean;
+    ErrorComponent?: any | boolean;
+    hideIcon?: boolean;
+    headerMessageId?: string;
+}
+
+function ValidationErrorSummary({
```

Comment:
We chatted a bit about this... but generally we should not duplicate to this level without first breaking shared stuff into a stand alone file first.... mainly since a good chunk of this is very easily shared.

Reply:
_Pending PR draft reference._